cmake/KokkosCore_config.h.in

```diff
@@ -54,6 +54,7 @@
 #cmakedefine KOKKOS_ENABLE_DUALVIEW_MODIFY_CHECK
 #cmakedefine KOKKOS_ENABLE_COMPLEX_ALIGN
 #cmakedefine KOKKOS_OPT_RANGE_AGGRESSIVE_VECTORIZATION
+#cmakedefine KOKKOS_ENABLE_VIEW_HOOKS
```

Comment:
Not sure that I want an option for that. In particular not a public option. In principle that would mean we need a lot of additional testing. Is there a good reason for this to be off?

Reply:
no real reason other that it is a new feature that affects pretty much everything.  We can make it always on if preferred.

Reply:
Please don't repeat the bad Trilinos habit of adding configure-time options that never get tested  : - )

Reply:
option removed 60b42a7d87599a9d2c2bfe913cb811b38a3b6f9f